.github/workflows/integration.yaml

```diff
@@ -0,0 +1,48 @@
+name: Integration
+on: [push, pull_request]
+env:
+  GALAXY_TEST_DBURI: 'postgres://postgres:postgres@localhost:5432/galaxy?client_encoding=utf8'
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7]
```

Comment:
```suggestion
        python-version: [2.7, 3.7]
```

Reply:
Test are still running, probably not needed.